modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -51,10 +55,12 @@ Default configuration
 | Message Broker | Description | Supported Protocols | Configuration Details 
 | Anypoint MQ | Supports asynchronous messaging with Anypoint MQ. | anypointmq | Anypoint MQ configurations are specified in the server object of the AsyncAPI specification. Supports `Message Listener` and `Subscribe` operations. 
 | Kafka | Supports asynchronous messaging with Apache Kafka. | kafka, kafka-secure | Kafka configurations are specified in the server object of the AsyncAPI specification. Supports `Message Listener` and `Subscribe` operations. 
-| Salesforce platform events | Allows implementing an Async API specification using the Salesforce connector. | salesforcepubsub | Connection details are available in the properties file. Supports `Message Listener` and `Subscribe` operations. 
-| Solace Pub/Sub+ | Supports the implementation of an Async API specification using the Solace connector. | solace | Connection details are available in the properties file. 
+| Salesforce platform events | Allows implementing an Async API specification using the Salesforce connector. | salesforcepubsub* | Connection details are available in the properties file. Supports `Message Listener` and `Subscribe` operations, and `Message Listener` and `Publish` operations.
+| Solace Pub/Sub+ | Supports the implementation of an Async API specification using the Solace connector. | solace | Connection details are available in the properties file. Supports `Message Listener` and `Publish` operations. 
 |===
 
++++*+++ You must use *salesforcepubsub* connector 1.0.15 or later.
```

Comment:
Correct connector name?

Reply:
Anypoint Connector for Salesforce Pub/Sub (Salesforce Pub/Sub Connector)

Reply:
OK! Thanks!